incubator/hnc/config/default/manager_auth_proxy_patch.yaml

```diff
@@ -0,0 +1,25 @@
+# This patch inject a sidecar container which is a HTTP proxy for the controller manager,
+# it performs RBAC authorization against the Kubernetes API using SubjectAccessReviews.
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+      - name: kube-rbac-proxy
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.4.0
+        args:
+        - "--secure-listen-address=0.0.0.0:8443"
+        - "--upstream=http://127.0.0.1:8080/"
+        - "--logtostderr=true"
+        - "--v=10"
```

Comment:
probably want to change this before releasing

Reply:
These are simply the defaults from kubebuilder 2.0, and I don't know enough about kube-rbac-proxy to know what the correct values are. I'm happy to add a TODO here, do you have any suggestions of what I should write (other than just "revisit before initial release")?